### ISSUE TITLE
Record live video, after more than 30 minutes, the audio will have Zizi sound, and then the sound will disappear;

### ISSUE THREAD

**Issue by zhudongyong** [ExistentialAudio/BlackHole#284]
Record live video, after more than 30 minutes, the audio will have Zizi sound, and then the sound will disappear;

**Comment by devinroth** [comment]
Use drift correction.

Devin Roth

**Comment by emilvlle** [comment]
Used Drift Correction
[12 min fii, even when using Drift.docx](https://github.com/ExistentialAudio/BlackHole/files/5911716/12.min.fii.even.when.using.Drift.docx)

**Comment by emilvlle** [comment]
Used drift correction and sound still drops at 12 min mark

**Comment by devinroth** [comment]
How exactly do you have it setup? Please add screenshots.

Devin Roth

**Comment by emilvlle** [comment]
[Uploading 12 min fizz, even when using Drift.docx…]()

**Comment by emilvlle** [comment]
i have attached the screenshots. hope this helps

**Comment by devinroth** [comment]
Okay and OS?

Devin Roth

**Comment by devinroth** [comment]
Please explain your setup so I can try to replicate it.

**Comment by zhudongyong** [comment]
It's ok

发自我的iPhone

> 在 2021年2月3日，下午1:48，Devin Roth <notifications@github.com> 写道：
> 
> ﻿
> Okay and OS?
> 
> Devin Roth

**Comment by lavint** [comment]
I have the same issue. After 30mins ish, the sound started to break and eventually dropped. I tried 3 times. Did you find a solution? I tried to use 16c and then 2c. Both have the same issue. Mac Air M1, Big Sur. I tried to use 48 Hz for all audios and then 44Hz for all audios. Nothing works.

<img width="455" alt="Screen Shot 2021-02-13 at 6 43 43 PM" src="https://user-images.githubusercontent.com/23566170/107864557-66aa0680-6e2b-11eb-9b67-e3a9e164254a.png">
<img width="480" alt="Screen Shot 2021-02-13 at 6 44 33 PM" src="https://user-images.githubusercontent.com/23566170/107864571-85100200-6e2b-11eb-97b7-95f32e89bd9c.png">

@devinroth will you be able to look into the issue?
